internal/engine/build_and_deployer_live_update_test.go

```diff
@@ -75,17 +75,18 @@ func runTestCase(t *testing.T, f *bdFixture, tCase testCase) {
 	f.assertContainerRestarts(tCase.expectDockerRestartCount)
 	assert.Equal(t, tCase.expectSyncletHotReload, f.sCli.UpdateContainerHotReload, "synclet hot reload")
 
-	id := manifest.ImageTargetAt(0).ID()
-	_, hasResult := result[id]
-	assert.True(t, hasResult)
-	assert.Equal(t, k8s.MagicTestContainerID, result.OneAndOnlyContainerID().String())
-
 	if tCase.expectK8sDeploy {
 		expectedYaml := "image: gcr.io/some-project-162817/sancho:tilt-11cd0b38bc3ceb95"
 		if !strings.Contains(f.k8s.Yaml, expectedYaml) {
 			t.Errorf("Expected yaml to contain %q. Actual:\n%s", expectedYaml, f.k8s.Yaml)
 		}
 		assert.Equal(t, tCase.expectSyncletDeploy, strings.Contains(f.k8s.Yaml, sidecar.SyncletImageName), "expected synclet-deploy = %t (deployed yaml was: %s)", tCase.expectSyncletDeploy, f.k8s.Yaml)
+	} else {
+		// Did a container build, expect result to have the container ID we operated on
```

Comment:
fwiw, i think DockerCompose builds also return a container ID

Reply:
yyyeahhh. i'll leave a comment.

Reply:
hmm actually I'm not sure that's relevant.